tests/rc/reader_utils_test.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from allennlp_models.rc.common.reader_utils import char_span_to_token_span
+
+
+@pytest.mark.parametrize(
+    "token_offsets, character_span, expected_result",
+    [
+        ([(0, 3), (3, 4), (4, 8)], (3, 7), ((1, 2), True)),
+        ([(0, 3), (3, 4), (4, 8)], (0, 7), ((0, 2), True)),
+        ([(0, 3), (3, 4), (4, 8)], (1, 7), ((0, 2), False)),
```

Comment:
This test is failing

Reply:
From my reading of it, all three examples should return `True`. Neither of them asks for offsets that are a precise match, so each of them should set the `error` flag.

Reply:
My intention was that the first two tests would be an exact match. I'm guessing I have the wrong endpoints though. Maybe `(3, 8)` and `(0, 8)` instead then?